test/images/scripts/rhel-atomic.bootstrap

```diff
@@ -4,7 +4,7 @@
 set -e
 
 url="http://cdn.stage.redhat.com/content/dist/rhel/atomic/7/7Server/x86_64/images/"
-prefix="rhel-atomic-cloud-7.2"
+prefix="rhel-atomic-cloud-7.2-[0-9\.]+.x86_64.qcow2"
```

Comment:
Also, this naming pattern seems to be outdated. Starting in June, the names look like this: `rhel-atomic-cloud-7.2.5.x86_64.qcow2` and `rhel-atomic-cloud-7.2.6-1.x86_64.qcow2`